src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/ServerRole/ServerRoleInfo.cs

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.SqlTools.ServiceLayer.ObjectManagement
+{
+    /// <summary>
+    /// a class for storing various server role properties
+    /// </summary>
+    public class ServerRoleInfo : SqlObject
+    {
+        public string? Owner { get; set; }
+        public string[]? Members { get; set; }
+        public string[]? Memberships { get; set; }
+        public bool IsFixedRole { get; set; }
```

Comment:
you can only add/remove members on fixed role, right?

Reply:
I'd recommend to move this to the view info since we are never going to set it in the UI.

Reply:
only add/remove members on fixed role - Yes
Makes sense I'll move it to view info.